openml-utils/src/test/java/com/feedzai/util/provider/AbstractProviderModelBaseTest.java

```diff
@@ -255,6 +268,33 @@ private void assertAllInstances(final List<List<Future<Integer>>> evaluationsLis
         return results;
     }
 
+    /**
+     * Classifies on of two dummy instances using a given model in parallel using a given {@link ExecutorService}.
+     * This method creates the same number of events for each dummy instance.
+     *
+     * @param model    Model to use to classify dummy instances.
+     * @param executor Executor to use to launch the threads that will classify each instance.
+     * @return A list of futures that will return the class of each instance.
```

Comment:
this is obtaining the class distribution, but the docs were copied from a classification request, so they are not 100% accurate